cli/cmd/summary.go

```diff
@@ -62,3 +73,86 @@ func checkFilePath(filename string) error {
 	dirPath := filepath.Dir(filename)
 	return os.MkdirAll(dirPath, 0755)
 }
+
+func generateSummaryTable() *schema.Table {
+	tableName := "cloudquery_sync_summary"
+	t := schema.Tables{{
+		Name:      tableName,
+		Transform: transformers.TransformWithStruct(&syncSummary{}),
```

Comment:
It would be nice if we could ensure that zero values for warnings and errors show up as zero, rather than null, in the table

Reply:
I think setting it to `Required: true` should fix it, but haven't tried yet

Reply:
I pushed an update for this as well